HISTORY.rst

```diff
@@ -5,6 +5,7 @@ History
 -------
 
 .. Insert new release notes below this line
+* Make the ``options`` argument to ``generate_barcode`` optional
```

Comment:
tests failing because reStructuredText wants a newline above this line

Reply:
Right. I wasn't sure whether you wanted the "Insert new release notes" line kept there or not. Will fix in a sec.